docs/versions

```diff
@@ -0,0 +1 @@
+updated-docs
```

Comment:
shouldn't this be a version (eg `v1.0`) for the docs

Reply:
Each line in `versions` is either an SDK release tag (e.g. `v.0.35.0`) or a branch name (e.g. `master`). It can also be a commit hash, because what deploy script essentially does is `checkout` each line and builds a website for that version. We use `updated-docs` for now, but I think later on version will look something like this:

```
master
v.0.37.0
v.0.36.0
```